### PR TITLE
Update Facebook to Graph API 3.2

### PR DESCRIPTION
**Pull request by dmpe**
Shall close https://github.com/python-social-auth/social-core/issues/279

### CODE REVIEW

**Comment by dmpe** [comment]
@ibolit  @tisdall @omab Hi, please could you test this new facebook update. 
It update Graph api to 3.2 and also adds profile picture in the return. 

```
sudo pip3 install git+https://github.com/dmpe/social-core.git@bug/facebook
```

Thank you

**Comment by ibolit** [comment]
Installing the patch to test it

File: social_core/backends/facebook.py [hunk 1]
```diff
@@ -59,6 +58,7 @@ def get_user_details(self, response):
         )
         return {'username': response.get('username', response.get('name')),
                 'email': response.get('email', ''),
+                'profile_picture': response.get('profile_picture', ''),
```
Comment (tisdall) [inline social_core/backends/facebook.py]: I don't think this should be added.  `get_user_details` is just to translate each authentication services response into a unified format.  The closest I could get to docs, though, was this:  https://github.com/omab/python-social-auth/issues/7#issuecomment-18087704

File: social_core/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '3.1.0'
+__version__ = '3.1.1'
```
Comment (tisdall) [inline social_core/__init__.py]: Not a big deal, but you should leave library versioning to the package maintainer.  They may want to include something that's a breaking change and then no 3.1.1 will ever be released, or they may release a 3.1.1 before ever merging your PR.

**Comment by dmpe** [comment]
@tisdall thanks for the feedback!

**Comment by tisdall** [comment]
So fast!  :)

I looked through the API docs and I don't see anything mentioned about OAuth in the changelogs so hopefully there's nothing else to change.

**Comment by ibolit** [comment]
We have tested it, everything seems to work just fine.

**Comment by ibolit** [comment]
I hate to be a nuisance, but can anyone merge this request?

**Comment by dmpe** [comment]
cc @omab

**Comment by tisdall** [comment]
To be clear, the current v2.9 API will stop working July 18, 2019 according to https://developers.facebook.com/docs/graph-api/changelog/

**Comment by omab** [comment]
Thanks!

**Comment by tisdall** [comment]
@ibolit @dmpe this is now released

**Pull request merged**